### PR TITLE
Make constructions in progress display the progress percentage.

### PR DESCRIPTION
**Pull request by ishtatann**
#### Summary

```SUMMARY:  Interface "Make constructions in progress display the progress percentage."```

#### Purpose of change

Small display change to show percentage of completion for construction tasks.

#### Describe the solution
Updated player_activity::get_progress_message to get the counter from the partial construction being worked on.

#### Testing
Loaded up a game and made a charcoal kiln - verified the progress display includes the percentage now.

### CODE REVIEW

**Comment by AMurkin** [comment]
This project has a specific code style. See [doc/CODE_STYLE.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.md).

File: src/player_activity.cpp [hunk 1]
```diff
@@ -104,8 +105,20 @@ cata::optional<std::string> player_activity::get_progress_message( const avatar
             type == activity_id( "ACT_CHOP_LOGS" ) ||
             type == activity_id( "ACT_CHOP_PLANKS" )
           ) {
+            const int percentage = ( ( moves_total - moves_left ) * 100 ) / moves_total;
```
Comment (BevapDin) [inline src/player_activity.cpp]:
```suggestion
            const int percentage = ( moves_total - moves_left ) * 100 / moves_total;
```

File: src/player_activity.cpp [hunk 2]
```diff
@@ -104,8 +105,20 @@ cata::optional<std::string> player_activity::get_progress_message( const avatar
...
+
             extra_info = string_format( "%d%%", percentage );
         }
+
+        if( type == activity_id( "ACT_BUILD" ) ) {
+            partial_con *pc = g->m.partial_con_at( g->m.getlocal( u.activity.placement ) );
+            if( pc ) {
+                int counter = std::min( pc->counter, 10000000 );
+                const int percentage = ( counter / 100000 );
```
Comment (BevapDin) [inline src/player_activity.cpp]:
```suggestion
                const int percentage = counter / 100000;
```

File: src/player_activity.cpp [hunk 3]
```diff
@@ -104,8 +105,20 @@ cata::optional<std::string> player_activity::get_progress_message( const avatar
...
           ) {
+            const int percentage = ( ( moves_total - moves_left ) * 100 ) / moves_total;
+
             extra_info = string_format( "%d%%", percentage );
         }
+
+        if( type == activity_id( "ACT_BUILD" ) ) {
+            partial_con *pc = g->m.partial_con_at( g->m.getlocal( u.activity.placement ) );
+            if( pc ) {
+                int counter = std::min( pc->counter, 10000000 );
```
Comment (BevapDin) [inline src/player_activity.cpp]: I wonder why the next variable is const, bu this one isn't?

Comment (ishtatann) [reply src/player_activity.cpp]: The original percentage line above was const.  I just copied that line and modified it.

I'm not a huge fan of const'ing everything under the sun so don't typically add it to variables I declare.

File: src/player_activity.cpp (hunk 1 above)
Comment (ishtatann) [reply src/player_activity.cpp]: I just moved this line I didn't add any additional parens.
Also, I prefer the explicit parens to the implicit depending on the order of operations.

**Pull request merged**